Maps.php

```diff
@@ -70,7 +70,7 @@
 		);
 	}
 
-	define( 'Maps_VERSION' , '4.3 alpha' );
+	define( 'Maps_VERSION' , '4.3' );
```

Comment:
Bunny is sad now.
![sadbunny](https://user-images.githubusercontent.com/1104078/27025852-3307c9ee-4f5c-11e7-96ab-49432b1b6339.jpg)